fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -654,7 +654,8 @@ void CommitBatchContext::setupTraceBatch() {
 
 			g_traceBatch.addAttach("CommitAttachID", tr.debugID.get().first(), debugID.get().first());
 		}
-		span.addParent(tr.spanContext);
+		// TODO - ljoswiak OK to add link here rather than parent? RE: addParent side effect
```

Comment:
Yeah I think this is the perfect place for a link, since this is where the batching is being done.

Reply:
OK marking done and removing TODO